### PR TITLE
#109 fix envvar settings

### PR DESCRIPTION
**Pull request by tobert**
#109 reports `OTEL_CLI_ATTRIBUTES` and `OTEL_CLI_SERVICE_NAME` aren't working. This PR will fix that.

### CODE REVIEW

**Review by emattiza — APPROVED** [review]
LGTM. If roundtrip on the env vars and the trace output is good, then it's doing its thing!

File: otlpserver/clievent.go
Comment (chuson) [inline otlpserver/clievent.go:114]: Would it be safer to split on the colon and then later truncate "_value"?

**Review by chuson — APPROVED** [review]
Up to you if you want to make changes where I commented, LGTM otherwise. 👍

File: otlpserver/clievent.go
Comment (tobert) [reply otlpserver/clievent.go:114]: I think they're about the same. I want this to be fragile and break tests if some new representation shows up. Also I originally used `strings.Cut` but switched to SplitN to defer raising the minimum go version for otel-cli. It looked better with Cut, I thought about changing it, but chose fewer lines of code in the end.

**Comment by tobert** [comment]
Thanks for the reviews. We need to get both of you on the approver list.

Superseded by #120.

**Pull request closed without merging**
